### ISSUE TITLE
Loss Function goes to "nan"

### ISSUE THREAD

**Issue by tamraouisebti** [omni-us/squeezedet-keras#29]

**Comment by bayesian-mind** [comment]
What was the solution to this problem? I am having similar issue where I am getting nan losses?

**Comment by tamraouisebti** [comment]
Hi,
The problem was with the initialization of the weights, when you use
ImageNet weights it goes to "nan". Initialize them to random or you can use
the trained model "kitti.h5".
Hope this helps, if you need something else just let me know.
